package.json

```diff
@@ -189,7 +189,7 @@
     "uuid": "^8.3.0",
     "victory-native": "^36.6.8",
     "vm-browserify": "^1.1.2",
-    "web3": "1.3.5"
+    "web3": "1.8.1"
```

Comment:
do we still need to pin or can we use range?

Reply:
No idea about the context, but looks like there was a habit of adding pinned packages at one point and we're just carrying it forward.  I'll add a `^`.

Reply:
also there's a resolution for web3 🙃  https://github.com/valora-inc/wallet/pull/3246/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R337